pom.xml

```diff
@@ -91,8 +91,12 @@
         <license.licensename>apache_v2</license.licensename>
 
         <!-- sonarcloud properties -->
-        <sonar.organization>axual</sonar.organization>
+        <sonar.organization>tonvanbart</sonar.organization>
```

Comment:
Should we rename this back to axual before merging?

Reply:
Yep